libraries/stm32f0xx/src/retarget.c

```diff
@@ -0,0 +1,34 @@
+#include "retarget.h"
```

Comment:
I would ideally like comments briefly explaining the behavior of these functions. It is not immediately obvious as to what is happening to someone who didn't actually write this code. Up to you though, this isn't a blocking issue.